README.md

```diff
@@ -1210,6 +1210,7 @@ _See also [Wikis](#wikis)_
 - [bepasty](https://bepasty-server.readthedocs.io/en/latest/) - A pastebin for all kinds of files. ([Source Code](https://github.com/bepasty/bepasty-server)) `BSD-2-Clause` `Python`
 - [bin](https://github.com/w4/bin) - a paste bin. `WTFPL/0BSD` `Rust`
 - [cryptonote](https://cryptonote.me/) - Simple open source web application that lets users encrypt and share messages that can only be read once. ([Source Code](https://github.com/alainmeier/cryptonote)) `MIT` `Ruby`
+- [dogbin](https://github.com/dogbin/dogbin) - The sexiest pastebin and url shortener ever. ([Demo](https://del.dog/) `MIT` `Kotlin`
```

Comment:
https://del.dog/ appears to be down.

Reply:
It got taken down by the author a few days ago. Pushed a commit to remove the preview.